KioskApplication/src/main/java/controller/PathfindingSidebarController.java

```diff
@@ -458,4 +498,28 @@ private void addWaypointBox() {
 //        addWaypointBoxIndex = waypointListView.getItems().indexOf(addWaypointBox);
 
     }
+
+
+    /**
+     * get the nearest node of required type to the default kiosk location
+     */
+    @FXML
+    private void handleButtonAction(ActionEvent e) throws  PathfinderException {
```

Comment:
We should move the map to the newly selected node on the correct floor when you press the button. Also maybe when the quick select button is pressed it clears the current path and current waypoints and adds the new waypoint as the only location. Up for discussion. It could also just immediately path find to the newly selected node.

Reply:
We can talk about this at the UI review tonight.

Reply:
sure

Reply:
I can work on getting the map to focus on a specific point on the map, the scroll panes are mildly fucky with cooridnates